local/build-binary

```diff
@@ -36,6 +36,37 @@ mkdir -p ${BUILD_DIR} ${INSTALL_DIR} ${DOWNLOAD_DIR}
 SOURCEDIR=$(cd ${2:-$(dirname $0)/../sources}; pwd -P)
 
 
+# ------------------------------------------------------------------------------
+# Setup ccache if enabled
+# ------------------------------------------------------------------------------
+if [[ "${USE_CCACHE}" == "1" ]]; then
+    echo "Setting up ccache..."
+
+    # Configure ccache (already installed in Docker image)
+    if command -v ccache &> /dev/null; then
+        ccache --version
+        ccache --max-size="${CCACHE_MAXSIZE:-2G}"
+        ccache --set-config compression=true
+        ccache --set-config compression_level=9
```

Comment:
Probably, no need to set up environment variables `CCACHE_COMPRESS=1` and `CCACHE_COMPRESSLEVEL=9` if you set these values here.